src/openzaak/config/templates/admin/config_detail.html

```diff
@@ -75,6 +75,45 @@ <h1>{% trans 'Open Zaak API dashboard' %}</h1>
             </div>
         </section>
 
+        <!-- External API config -->
+        <section class="module">
+            <table class="table">
+                <caption>External services</caption>
+                <thead>
+                    <tr>
+                        <th scope="col">{% trans "Service" %}</th>
+                        <th scope="col">{% trans "Type" %}</th>
+                        <th scope="col">{% trans "Url" %}</th>
+                        <th scope="col">{% trans "NLX" %}</th>
+                        <th scope="col">{% trans "OAS" %}</th>
+                        <th scope="col">{% trans "Authorization" %}</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for service in external_services %}
+                    <tr>
+                        <td>{{ service.label }}</td>
+                        <td>{{ service.get_api_type_display }}</td>
+                        <td>{{ service.api_root|urlize }}</td>
+                        <td>{{ service.nlx|urlize }}</td>
+                        <td>{{ service.oas|urlize }}</td>
```

Comment:
```suggestion
                        <td><a href="{{ service.oas|urlize }}" target="_blank">{% trans "Externe link" %}</td>
```

This saves some space.